streams/src/test/java/org/apache/kafka/streams/integration/InternalTopicIntegrationTest.java

```diff
@@ -71,6 +71,7 @@
 /**
  * Tests related to internal topics in streams
  */
+@SuppressWarnings("deprecation")
```

Comment:
Ok, I see now that there are a lot of tests using the deprecated methods. We should absolutely migrate them all to the new APIs (of course we will be forced to once the deprecated ones are removed), but it's ok with me, and maybe even slightly preferable, to do this in a followup PR. That way we can keep this PR focused on the changes themselves and just the tests for the new APIs.

Can you file a ticket to migrate all of the remaining tests over to the new APIs and remove the warning suppression? Then you can start working on that after this PR is merged, or in parallel with the review but on a separate branch

Reply:
Thanks @ableegoldman for the feedback. I will work on your recommendations on the Javadocs and I will create the JIRA issue to track the migration of integration and unit tests to the new APIs.

As you have seen, the amount of tests are significant so that one deserves its own JIRA issue. 

I really appreciate all your support and guidance throughout the implementation of this KIP.

Reply:
@ableegoldman I have created JIRA ticket KAFKA-12994 to track the tasks of migrating the tests to the new APIs and then removing the deprecation warnings.

https://issues.apache.org/jira/browse/KAFKA-12994